mobile/src/main/java/com/gelakinetic/mtgfam/helpers/tcgp/MarketPriceInfo.java

```diff
@@ -0,0 +1,193 @@
+/*
+ * Copyright 2018 Adam Feinstein
+ *
+ * This file is part of MTG Familiar.
+ *
+ * MTG Familiar is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * MTG Familiar is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with MTG Familiar.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.gelakinetic.mtgfam.helpers.tcgp;
+
+import com.gelakinetic.mtgfam.helpers.tcgp.JsonObjects.ProductDetails;
+import com.gelakinetic.mtgfam.helpers.tcgp.JsonObjects.ProductMarketPrice;
+
+public class MarketPriceInfo {
+
+    public enum PriceType {
+        LOW,
+        MID,
+        HIGH,
+        MARKET;
+
+        /**
+         * Convert a price type to an integer
+         * This must match the values in trade_option_entries and trade_option_values
+         *
+         * @return The integer representation of the type
+         */
+        public int toInt() {
```

Comment:
`ordinal()` returns the index of the item

Reply:
Neat, updated to use that